kommunicateui/src/main/res/layout/mobicom_message_list.xml

```diff
@@ -453,6 +454,15 @@
                                 android:background="@color/apploizc_transparent_color"
                                 android:src="@drawable/camera" />
 
+                            <ImageButton
+                                android:id="@+id/idMultiSelectGalleryButton"
+                                android:layout_width="20dp"
+                                android:layout_height="18dp"
+                                android:layout_marginLeft="30dp"
+                                android:layout_marginStart="30dp"
+                                android:background="@color/apploizc_transparent_color"
```

Comment:
@shubhamtewari correct spelling of applozic

Reply:
corrected